<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mentor</class>
 <widget class="QMainWindow" name="Mentor">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1228</width>
    <height>805</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>11</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MENTOR</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 11pt &quot;MS Shell Dlg 2&quot;;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
        stop:0 rgba(173, 216, 230, 255), 
        stop:1 rgba(255, 255, 255, 255));
};

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>95</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
    padding: 10px;
    font-size: 16px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #ccc;
    border-radius: 10px;
    margin-bottom: 10px;
}

QLabel:hover {
    background-color: #007B9E;
}
QLabel:pressed {
    background-color: #005f73;
}border-radius: 10px;
background-color: #008CBA;</string>
      </property>
      <property name="text">
       <string>MENTOR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_mentor_preference">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0,0">
       <item row="2" column="0">
        <widget class="QPushButton" name="PB_mentor_preferences">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 11pt &quot;MS Shell Dlg 2&quot;;
    background-color: #008CBA;
    color: white;
    padding: 10px;
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    border: none;
    border-radius: 10px;
    margin: 5px;
}
QPushButton:hover {
    background-color: #007B9E;
}
QPushButton:pressed {
    background-color: #005f73;
}
</string>
         </property>
         <property name="text">
          <string>Main Menu</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/news.svg</normaloff>icons/news.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="PB_Mentor_Find">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 11pt &quot;MS Shell Dlg 2&quot;;
    background-color: #008CBA;
    color: white;
    padding: 10px;
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    border: none;
    border-radius: 10px;
    margin: 5px;
}
QPushButton:hover {
    background-color: #007B9E;
}
QPushButton:pressed {
    background-color: #005f73;
}
</string>
         </property>
         <property name="text">
          <string> Find     </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/search-user.svg</normaloff>icons/search-user.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="PB_mentor_all_interviews">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 11pt &quot;MS Shell Dlg 2&quot;;
    background-color: #008CBA;
    color: white;
    padding: 10px;
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    border: none;
    border-radius: 10px;
    margin: 5px;
}
QPushButton:hover {
    background-color: #007B9E;
}
QPushButton:pressed {
    background-color: #005f73;
}
</string>
         </property>
         <property name="text">
          <string>All Interviews</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/profile.svg</normaloff>icons/profile.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBox_Mentor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>59</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    padding: 10px;
    font-size: 14px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #ccc;
    border-radius: 10px;
    margin-bottom: 10px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 1px;
    border-left-color: #ccc;
    border-left-style: solid;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}
</string>
         </property>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesinin tamamına katılması uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi ilk IT eğitimi alıp ITPH a yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi ingilizce eğitimi alıp ITPH a yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VIT projesi kapsamında direkt ITPH a yönlendirilmesi uygun olur.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Direkt bireysel koçluk ile işe yönlendirilmesi uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bir sonraki VIT projesine katilmasi daha uygun olur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Başka bir sektöre yönlendirilmeli</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diger</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    padding: 10px;
    font-size: 16px;
    border: 2px solid #ccc;
    border-radius: 10px;
    margin-bottom: 10px;
}

QPushButton:hover {
    background-color: #007B9E;
}
QPushButton:pressed {
    background-color: #005f73;
}
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="PB_mentor_exit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 11pt &quot;MS Shell Dlg 2&quot;;
    background-color: #008CBA;
    color: white;
    padding: 10px;
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    border: none;
    border-radius: 10px;
    margin: 5px;
}
QPushButton:hover {
    background-color: #007B9E;
}
QPushButton:pressed {
    background-color: #005f73;
}
</string>
         </property>
         <property name="text">
          <string>Exit      </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/power.svg</normaloff>icons/power.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>883</width>
        <height>450</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>883</width>
        <height>450</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
	
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
    background-color: #f9f9f9;
    alternate-background-color: #e0e0e0;
    gridline-color: #ccc;
    border-radius: 10px;
}
QHeaderView::section {
    background-color: #008CBA;
    color: white;
    padding: 5px;
    font-size: 14px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
}
</string>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>154</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>38</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>38</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Surname</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Interview Date</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mentor</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IT Knowledge</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Intensity</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comments</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
